src/components/SearchAndFilter/FilterPanelSection/FilterPanelSection.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Chip from "../../Chip";
+
+import "./filter-panel-section.scss";
+
+const FilterPanelSection = ({ data }) => (
+  <div className="filter-panel-section">
+    {data?.heading && (
+      <h3 className="filter-panel-section__heading">{data.heading}</h3>
+    )}
+    {data?.chips?.map((chip, i) => (
+      <Chip lead={chip.lead} value={chip.value} key={i} />
```

Comment:
If the order of the chips in the `data` changes then `key` will become invalid and trigger a re-render. It'll be a bit more performant if the key is tied to the representation of the chip, so maybe...

```suggestion
      <Chip lead={chip.lead} value={chip.value} key={chip.lead+chip.value} />
```